code/__DEFINES/misc.dm

```diff
@@ -367,7 +367,7 @@
 #define INVESTIGATE_BOMB "bombs"
 
 // The SQL version required by this version of the code
-#define SQL_VERSION 5
+#define SQL_VERSION 6
```

Comment:
but why.

Reply:
If it’s why the version was changed, because this PR requires a change in the SQL backend

If it’s why does it need SQL, save files are fucking awful and barely work